demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c

```diff
@@ -105,6 +105,7 @@ typedef struct ProvisionedState_t
     CK_OBJECT_HANDLE xPublicKey;
     uint8_t * pucDerPublicKey;
     uint32_t ulDerPublicKeyLength;
+    char * pcIdentifier;
```

Comment:
It would be nice to know what this identifier identifies...  Is it the cert DN or serial number or signature or something else?

Reply:
hi yes, it looks like this will be updated to contain the token label.

In the specific case of the ECC608a, this happens to be the ECC608a serial number, which is required (with the template/configs specified by this project) to be used as the common name of the certificate.

I will add some comments here.